ui/helpers/constants/design-system.js

```diff
@@ -207,3 +209,5 @@ export const RESIZE = {
   INITIAL: 'initial',
   INHERIT: 'inherit',
 };
+
+export const BREAKPOINTS = ['base', 'sm', 'md', 'lg'];
```

Comment:
Adding breakpoints const to be used for responsive props generation. I can also see this possibly being used for a useMediaQuery hook in future.